fdbserver/TLogServer.actor.cpp

```diff
@@ -969,7 +985,10 @@ ACTOR Future<Void> popDiskQueue(TLogData* self, Reference<LogData> logData) {
 	return Void();
 }
 
-ACTOR Future<Void> updatePersistentData(TLogData* self, Reference<LogData> logData, Version newPersistentDataVersion) {
+ACTOR Future<Void> updatePersistentData(TLogData* self,
```

Comment:
Since every caller of this method waits on the FlowMutex beforehand, would it make more sense to just acquire the mutex inside this actor by moving the
```
			FlowMutex::Lock lock = wait(self->persistentDataCommitLock.take());
```
in here?

Reply:
Oh I see, the lock is not released inside here.